### PR TITLE
added yosemite model and world

### PR DESCRIPTION
**Pull request by ahcorde**
Added yosemite

 - model
 - example world

yosemite

co-author: @tfoote

### CODE REVIEW

**Comment by tfoote** [comment]
Please test this out as is for feedback.  Before it's ready to merge we're going to move the models to be hosted at gazebosim.org/models so that the model data isn't stored here in this repository. But before we finalize the model upload there we'd like to get any feedback from reviews here.

**Comment by TSC21** [comment]
@tfoote these are great! Please host them on the models site so we can import them from there. Some doc updates in our Devguide with some references to these are welcomed!

**Comment by hamishwillee** [comment]
FYI, PX4 docs for this would probably be here: http://dev.px4.io/master/en/simulation/gazebo.html

In the very near future there will be a section on selecting the world, which will be possible using a make target OR an environment variable. So pointing people to where they can find out about worlds might be cool.

**Comment by RicardoM17** [comment]
@tfoote @ahcorde it would also be interesting if you could leave some guide or steps that you followed should someone want to replicate this for their custom projects (for example me :stuck_out_tongue: ).

**Comment by ahcorde** [comment]
@RicardoM17 , you only need to compile the package and run the example world

```
rosrun gazebo_ros gazebo worlds/yosemite.world
```

myabe you need to run it with the full path

**Comment by RicardoM17** [comment]
@ahcorde I meant the steps for one to create their own maps. Basically the important parts here is how did you get the elevation files etc.

But this is slightly offtopic not sure if we should discuss this here. But I'm certainly interested and it should be in the docs too imo. There's gazebo docs on how to create this but they are quite outdated (links lead to dead sites for example)

**Comment by RicardoM17** [comment]
@tfoote @ahcorde any updates on this last comment from me?

> @ahcorde I meant the steps for one to create their own maps. Basically the important parts here is how did you get the elevation files etc.

**Comment by ahcorde** [comment]
```
    os.environ["PX4_HOME_LAT"] = "37.7332531";
    os.environ["PX4_HOME_LON"] = "-119.5616378";
    os.environ["PX4_HOME_ALT"] = "2800.4";
```

**Comment by RicardoM17** [comment]
@ahcorde that is just the environment variables to use your map.

What I'm asking for is what steps did you follow to create the elevation and colour mapping, i.e. the following files:

models/yosemite/media/yosemite_elevation.tif 

models/yosemite/media/textures/yosemite_color.png

If you could share them that would be great.

For reference here is what is the correct approach according to the Gazebo docs but unfortunately many of the links are broken.

http://gazebosim.org/tutorials?tut=dem&cat=build_world

**Comment by TSC21** [comment]
> @ahcorde that is just the environment variables to use your map.
> 
> What I'm asking for is what steps did you follow to create the elevation and colour mapping, i.e. the following files:
> 
> models/yosemite/media/yosemite_elevation.tif
> 
> models/yosemite/media/textures/yosemite_color.png
> 
> If you could share them that would be great.
> 
> For reference here is what is the correct approach according to the Gazebo docs but unfortunately many of the links are broken.
> 
> http://gazebosim.org/tutorials?tut=dem&cat=build_world

@RicardoM17 the above was an answer to https://github.com/PX4/sitl_gazebo/pull/405#issuecomment-603144109.

**Comment by RicardoM17** [comment]
> > @ahcorde that is just the environment variables to use your map.
> > What I'm asking for is what steps did you follow to create the elevation and colour mapping, i.e. the following files:
> > models/yosemite/media/yosemite_elevation.tif
> > models/yosemite/media/textures/yosemite_color.png
> > If you could share them that would be great.
> > For reference here is what is the correct approach according to the Gazebo docs but unfortunately many of the links are broken.
> > http://gazebosim.org/tutorials?tut=dem&cat=build_world
> 
> @RicardoM17 the above was an answer to [#405 (comment)](https://github.com/PX4/sitl_gazebo/pull/405#issuecomment-603144109).

Whoops my bad then (though I had no way of knowing :laughing: )

**Comment by ahcorde** [comment]
I updated the PR related with new worlds. PR is merged https://bitbucket.org/osrf/gazebo_models/pull-requests/360/added-diux-wordls/diff

**Comment by tfoote** [comment]
> But this is slightly offtopic not sure if we should discuss this here. But I'm certainly interested and it should be in the docs too imo. There's gazebo docs on how to create this but they are quite outdated (links lead to dead sites for example)

@RicardoM17  The way that I generated this content is documented here: https://github.com/osrf/uav_testing/blob/master/README.md

**Comment by RicardoM17** [comment]
Thanks @tfoote I will have a look soon and see if I can replicate on my side for my use case :)

File: worlds/yosemite.world [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        </clouds>
+      </sky>
+    </scene>
+    <!-- A global light source -->
+    <include>
+      <uri>model://sun</uri>
+    </include>
+    <include>
+      <uri>model://asphalt_plane</uri>
+    </include>
```
Comment (Jaeyoung-Lim) [inline worlds/yosemite.world]: Is the asphalt_plane required? It looks a little strange with the rest of the terrain: [video](https://youtu.be/7bAz36IsBko)

**Review by Jaeyoung-Lim — CHANGES_REQUESTED** [review]

File: worlds/yosemite.world (hunk 1 above)
Comment (TSC21) [reply worlds/yosemite.world]: I don't think it makes sense.

**Review by TSC21 — CHANGES_REQUESTED** [review]
@ahcorde please remove the `asphalt_plane` and this is good to go. Thank you for the contribution!

**Comment by ahcorde** [comment]
:+1:

**Review by Jaeyoung-Lim — APPROVED** [review]
@ahcorde Thanks!

Tested, and it looks good!

**Review by TSC21 — APPROVED** [review]

File: worlds/yosemite.world (hunk 1 above)
Comment (tfoote) [reply worlds/yosemite.world]: It was there to support our demo allowing parametric launching of N vehicles and when spawning them it was much easier to have a flat plane to target. Removing it's no problem, you just have to be more careful about where you spawn is flat enough.

**Pull request merged**